draft-ietf-quic-qlog-main-schema.md

```diff
@@ -651,69 +650,150 @@ JSON serialization example:
 
 ## Data {#data-field}
 
-An event's "data" field is a generic object. It contains the per-event metadata and its
-form and semantics are defined per specific sort of event. For example, data
-field value definitions for QUIC and HTTP/3 can be found in {{QLOG-QUIC}} and
-{{QLOG-H3}}.
+An event's "data" field is a generic key-value map (e.g., JSON object). It
```

Comment:
Is there any danger that mentioning JSON object here could confusing people about what serialization formats are allowed?

Reply:
I thought about it, but I dont' think so... we discuss this in detail in the serialization section and JSON is the main provided option there, so I think it's fine to have it as an example here.

Reply:
agree